Cascading deletion of `OpenIddictAuthorizations ` and `OpenIddictTokens`.

maliming opened the pull request:
Resolve #21024

maliming committed:
Cascading deletion of `OpenIddictAuthorizations ` and `OpenIddictTokens`.

Resolve #21024

maliming committed:
Update template project migrations.

EngincanV commented:
I added the **breaking-change** label so as not to forget to mention this in the migration guide. (developers need to create a new migration)

maliming commented:
> I added the **breaking-change** label so as not to forget to mention this in the migration guide. (developers need to create a new migration)

hi @EngincanV 

The changes are unnecessary, so I have reverted them.
